Ciphers/AffineCipher.js

```diff
@@ -0,0 +1,88 @@
+
+/**
+ * @description - The affine cipher is a type of monoalphabetic substitution cipher, where each letter in an alphabet is mapped to    its numeric equivalent, encrypted using a simple mathematical function, and converted back to a letter
+ * @see - [wiki](https://en.wikipedia.org/wiki/Affine_cipher)
+ */
+
+// Default key for Affine Cipher
+const key = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+/**
+ * Fix result for negative value in modulas equation
+ * @param {Number} n - Constant number
+ * @param {Number} m - Modulos value
+ * @return {Number} Return n mod m
+ */
+function mod (n, m) {
+  return ((n % m) + m) % m
+}
+
+/**
+ * Modular multiplicative inverse
+ * @param {Number} a - A coefficient
+ * @param {Number} m - Modulos value
+ * @return {Number} Return modular multiplicative inverse of coefficient a and modulos m
+ */
+function inverseMod (a, m) {
+  for (let x = 1; x < m; x++) {
+    if (mod(a * x, m) === 1) return x
+  }
+}
+
+/**
+ * Argument validation
+ * @param {String} str - String to be checked
+ * @param {Number} a - A coefficient to be checked
+ * @param {Number} b - B coefficient to be checked
+ * @return {Boolean} Result of the checking
+ */
+function isCorrectFormat (str, a, b) {
+  if (typeof a !== 'number' || typeof b !== 'number') {
+    throw new TypeError('Coefficient a, b should be number')
+  }
+
+  if (typeof str !== 'string') {
+    throw new TypeError('Argument str should be String')
+  }
+
+  return true
+}
+
+/**
+ * Encrypt a Affine Cipher
+ * @param {String} str - String to be encrypted
+ * @param {Number} a - A coefficient
+ * @param {Number} b - B coefficient
+ * @return {String} result - Encrypted string
+ */
+function encrypt (str, a, b) {
+  let result = ''
+  if (isCorrectFormat(str, a, b)) {
+    for (let x = 0; x < str.length; x++) {
+      if (key.indexOf(str[x].toUpperCase()) === -1) result += '-1' + ' '
+      else result += mod(a * key.indexOf(str[x].toUpperCase()) + b, 26) + ' '
```

Comment:
Consider using `str[x].toUpperCase().charCodeAt(0) - ("A").charCodeAt(0)`. This runs in constant time whereas `key.indexOf` runs in linear time in the size of the alphabet.

Reply:
Thank you for reviewing my code. I have replace indexOf with your recommendation